src/pages/en/guides/client-side-scripts.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Scripts and Event Handling
+description: How to add client-side interactivity to Astro components using native browser JavaScript APIs.
+layout: ~/layouts/MainLayout.astro
+i18nReady: false
+---
+
+You can add interactivity to your Astro components without [using a UI framework](/en/core-concepts/framework-components/) like React, Svelte, Vue, etc. using standard HTML `<script>` tags. This allows you to send JavaScript to run in the browser and add functionality to your Astro components.
+
+## Using `<script>` in Astro
+
+In `.astro` files, you can add client-side JavaScript by adding one (or more) `<script>` tags.
+
+In this example, adding the `<Hello />` component to a page will log a message to the browser console.
+
+```astro title="src/components/Hello.astro"
+<h1>Welcome, world!</h1>
+
+<script>
+  console.log('Welcome, browser console!');
+</script>
+```
+
+### Script bundling
+
+By default, `<script>` tags are processed by Astro.
+
+- Any imports will be bundled, allowing you to import local files or Node modules.
+- The processed script will be injected into your page’s `<head>` with [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).
+- TypeScript is fully supported, including importing TypeScript files.
+- If your component is used several times on a page, the script will only be included once.
+
+```astro title="src/components/Example.astro"
+<script>
+  // Processed! Bundled! TypeScript-supported!
+  // Importing local scripts and Node modules works.
+</script>
+```
+
+To avoid bundling the script, you can use the `is:inline` directive.
+
+```astro title="src/components/InlineScript.astro" "is:inline"
+<script is:inline>
+  // Will be rendered into the HTML exactly as written!
+  // Local imports are not resolved and will not work.
+</script>
+```
+
+:::note
+Adding `type="module"` or any other attribute to a `<script>` tag will disable Astro's default bundling behavior, treating the tag as if it had an `is:inline` directive.
```

Comment:
You _can_ add `src` and take advantage of bundling (as we show below) so tweaking this:

```suggestion
Adding `type="module"` or any attribute other than `src` to a `<script>` tag will disable Astro's default bundling behavior, treating the tag as if it had an `is:inline` directive.
```